native/src/components/PoiDetails.tsx

```diff
@@ -123,9 +124,18 @@ const PoiDetails = ({ poi, feature, language }: PoiDetailsProps): ReactElement =
           <HorizontalLine />
         </>
       )}
+      <>
+        <OpeningHours
+          language={language}
+          openingHours={openingHours}
+          isCurrentlyOpen={isCurrentlyOpen}
+          isTemporarilyClosed={temporarilyClosed}
+        />
+        {((openingHours && openingHours.length > 0) || temporarilyClosed) && <HorizontalLine />}
```

Comment:
Optional: Does it make sense to move the horizontal line to the collapsible item/OpeningHours components. Basically we render the horizontal line whenever we render a collapsible item or some other information, right?

Reply:
but the downside is you have to replicate it, but okay